streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -470,6 +475,8 @@ public void shouldRejectNonExistentStoreName() {
                 exception.getMessage(),
                 is("Cannot get state store no-table because no such store is registered in the topology.")
             );
+        } finally {
+            CLUSTER.deleteAllTopicsAndWait(0L);
```

Comment:
How about moving this cleanup to ```@after```?

Reply:
Sure, I can do that.

Thanks for the review!